src/libraries/geomorphology/geomorphology.0875.cc

```diff
@@ -43,28 +42,27 @@ void sky_view_factor(Matrix<double> *sky, long N, T_INIT *UV, Matrix<double> *in
     // Computation of the matrix with the angles of the direction
     alfa.reset(new Matrix<double>{2*input->nrh-1,2*input->nch-1});
     *alfa = (double)novalue; //initialisation with novalue
+
+#pragma omp parallel for private(i, j)
```

Comment:
did you try to `fuse` the two loops? did you test different `schedule` (`guided`, `dynamic`) to see if they perform better than the `static` which is the default?

Reply:
No I didn't.